### ISSUE TITLE
How to deal with only one pheno input for Conditional Template Creation?

### ISSUE THREAD

**Issue by kvttt** [voxelmorph/voxelmorph#460]
**Task** (what are you trying to do/register?)

I want to implement `ConditionalTemplateCreation` on PyTorch and test it on my own data of MRI scans of size [160, 192, 160] normalized to 0 to 1. Currently I only have age (as a continuous variable, normalized to 0 to 1) as the pheno attribute. And I am wondering how to set up the model. Will the model work with only age as its pheno input?

**What have you tried**

I have build the model in PyTorch with the same setup as following:

https://github.com/voxelmorph/voxelmorph/blob/7cf611ab283b22424d82650d6e17d49b986e7981/scripts/tf/train_cond_template.py#L140-L150

Specifically, I have set `inshape` to [160, 192, 160], `pheno_input_shape` to 1, `src_feats` and `trg_feats` to 1 and left the rest of the arguments as default.

**Details of experiments**

I used MSE loss, and the same optimizer setting. I trained the model till convergence. But the model doesn't seem to be learning anything. For visualization of the result I got, see the images I uploaded to #450. It looks like the decoded pheno attribute (i.e. the residual) is still just a volume of random noise. I am not sure what is going on. I think @adalca mentioned in #450 that for your experiment two pheno attributes are used (i.e. age and sex). So I am guessing it might be the case that the model does not work with single pheno attribute. 

Thanks in advance for your help!
